src/api/getWithdrawals.js

```diff
@@ -1,6 +1,29 @@
 const post = require('../lib/dvf/post-authenticated')
 const validateAssertions = require('../lib/validators/validateAssertions')
 
```

Comment:
If we don't have the `withdrawalBalanceReaderContractAddress` set, it's going to use the old method.

Reply:
> To test the usage of the contract I added the line above into [this file](https://github.com/DeversiFi/dvf-client-js/blob/dev/src/api/getUserConfig.js#L7)
> 
> ```
> exchangeUserConf.DVF.withdrawalBalanceReaderContractAddress = "0x7ce6cDB1AF464B32b9F827697201184d7895D660"
> ```
> 
> So I'm assuming we need to add this configuration on each env right @plutoegg @AliMeer

You're right that we will have a different contract on ropsten vs mainnet. I suggest adding it in here so that users of the library don't need to. Ideally it would automatically select for each environment which contract to use: https://github.com/DeversiFi/dvf-client-js/blob/master/src/config.js

Reply:
@plutoegg the contract addresses usually comes from the user config call to the backend on this line
https://github.com/DeversiFi/dvf-client-js/blob/dev/src/api/getUserConfig.js#L4

So I don't think the user that is using the lib will have to add the contract address, isn't that a backend or db configuration?

Reply:
Its true that we could provide it there, but i think its really just a client side / UI helper. I think it can just be hardcoded in this library since it wont change, and select the correct one for the correct network. Either way, i'm happy with whatever solution you think is best, but we should make it so that the user who just picks up this client library doesnt need to add it manually with `withdrawalBalanceReaderContractAddress`

Reply:
@plutoegg I understand your point, the reason why I was expecting it from the backend is because our current call to `/v1/trading/r/getUserConf` returns this

```
{
  exchangeUserConf: {
    DVF: {
      defaultFeeRate: 0.002,
      deversifiAddress: '0xc0e9bd267626e2393ddaf520ae5df27ef6c533bd',
      starkExContractAddress: '0x5783323064dDa4A1ebe62defFeF46750BD2C560c',
      exchangeSymbols: [Array],
      tempStarkVaultId: 1,
      minDepositUSDT: 1
    },
    tokenRegistry: { ETH: [Object], USDT: [Object], ZRX: [Object], BTC: [Object] },
    isRegistered: true,
    ethAddress: '0x965fd011aeedbdd9aadf1421781f0a1bcf3a5e2e'
  }
}
```

so since `starkExContractAddress` is there I thought that `withdrawalBalanceReaderContractAddress ` could be too.

Reply:
@plutoegg

Reply:
@thiagokroger OK, lets add it to the backend getConf call. I'll create all the contracts now and add them for the different environments.